security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,20 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-2-0"]
+== OpenShift Compliance Operator 1.2.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.2.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:4245[RHBA-2023:4245 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-2-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The CIS {product-title} 4 Benchmark v1.4.0 profile is now available for platform and node applications. To locate the CIS {product-title} v4 Benchmark, go to  link:https://www.cisecurity.org/benchmark/kubernetes[CIS Benchmarks] and click *Download Latest CIS Benchmark*, where you can then register to download the benchmark.
+
+* Additional clarity for auditing security context constraints (SCC) rules is now available for the `scc-limit-container-allowed-capabilities` rule.
```

Comment:
I realized this is an enhancement, not a traditional "bug" so any time we improve rules moving forward I want to document them as features and enhancements.